### DISCUSSION TITLE
AppImage Run issue

### DISCUSSION THREAD

**Discussion by rahidulislam** [linuxdeepin/developer-center#3496] [Troubleshooting | 故障排查]
Appimage format does not run.
![image](https://user-images.githubusercontent.com/33887458/202404456-a12a2603-fc04-46fb-93e6-f78699bd91d2.png)

**Comment by BLumia** [comment]
Is this v23-only or also exists on v20?

**Comment by Turbovix** [comment]
First I want to say that I love Deepin.
I am using OS version 20.7.1 and the AppImage of Balena Etcher(V. 1.10.2) downloaded from the developer's website does not open. However the one downloaded (V 1.7.9) from the Deepin store works perfectly.

**Reply by BLumia** [reply]
You can try upgrade deepin first (the most recent version is v20.8), if that still happen after upgrade, please try execute the program from commandline (with deepin-terminal for example) and see if there are any log/output so we can look into it. Thanks!
